docs/wagtail-pages.md

```diff
@@ -0,0 +1,207 @@
+        ('text_introduction', molecules.TextIntroduction()),
+    ], blank=True)
+```
+
+**Insert link to StreamField docs here.**
+
+## Panels
+
+Editor panels define how the page's [fields](#fields) and [StreamFields](#streamfields) will be organized for content editors.
+
+The base Wagtail `Page` class and the [`v1.models.base.CFGOVPage` class](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/base.py#L60) 
+defines a specific set of panels to which all fields should be added:
+
+- `content_panels`. 
+  These fields appear on the "General Content" tab when editing a page. 
+  For page body content.
+- [`sidefoot_panels`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/base.py#L114-L116).
+  These fields appear on the "Sidebar" tab when editing a page. 
+- [`settings_panels`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/base.py#L118-L125).
+  Appears on the "Configuration" tab when editing a page. 
+  Page configuration such as the categories, tags, scheduled publishing, etc.
+
+Most fields will simply require a `FieldPanel` to be added to one of the sets of panels above. 
+StreamFields will require a `StreamFieldPanel`. 
+See the [Wagtail documentation for additional, more complex panel options](https://docs.wagtail.io/en/v1.13.4/topics/pages.html#editor-panels).
+
+For example, in our `BrowsePage` (used in the [database fields example above](#fields),
+[the `secondary_nav_exclude_sibling_pages` `BooleanField` is added to the `sidefoot_panels` as a `FieldPanel`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/browse_page.py#L63):
+
+```python
+from django.db import models
+from wagtail.wagtailadmin.edit_handlers import FieldPanel
+
+from v1.models.base import CFGOVPage
+
+
+class BrowsePage(CFGOVPage):
+    secondary_nav_exclude_sibling_pages = models.BooleanField(default=False)
+
+    # …
+
+    sidefoot_panels = CFGOVPage.sidefoot_panels + [
+        FieldPanel('secondary_nav_exclude_sibling_pages'),
+    ]
+```
+
+In out `LandingPage` (used in the [StreamFields example above](#streamfields), 
+[the `header` StreamField is added to the `content_panels` as a `StreamFieldPanel`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/landing_page.py#L31):
+
+```python
+from wagtail.wagtailadmin.edit_handlers import StreamFieldPanel
+from wagtail.wagtailcore.fields import StreamField
+
+from v1.atomic_elements import molecules
+from v1.models import CFGOVPage
+
+
+class LandingPage(CFGOVPage):
+    header = StreamField([
+        ('hero', molecules.Hero()),
+        ('text_introduction', molecules.TextIntroduction()),
+    ], blank=True)
+
+    # …
+
+    content_panels = CFGOVPage.content_panels + [
+        StreamFieldPanel('header'),
+        # …
+    ]
+```
+
+## Parent / child page relationships
+
+Wagtail provides two attributes to page models that enable restricting the types of subpages or parent pages a particular page model can have. 
```

Comment:
Link to [Wagtail docs](https://docs.wagtail.io/en/v1.13.4/topics/pages.html#parent-page-subpage-type-rules) here?